config/nrfconnect/chip-module/Kconfig

```diff
@@ -250,6 +246,8 @@ endif # CHIP_FACTORY_DATA_BUILD
 config CHIP_FACTORY_RESET_ERASE_NVS
 	bool
 	default y
+	depends on !CHIP_CRYPTO_PSA
```

Comment:
You can use it with PSA as long as you don't migrate DAC key so this is not needed.